bin/run-wp-unit-tests.sh

```diff
@@ -32,8 +29,9 @@ fi
 
 # Run PHPUnit tests
 if [[ $DOCKER = "true" ]]; then
-	npm run test-php || exit 1
-	npm run test-unit-php-multisite || exit 1
+	docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm composer run-script lint
```

Comment:
What's the reason for calling those commands directly rather than continue using `npm run *`?

Reply:
> What's the reason for calling those commands directly rather than continue using `npm run *`?

I guess I assumed if we weren't installing dependencies, we shouldn't run npm scripts. In retrospect, it's probably not strictly a problem. Overall, it's part of a move of "we don't need NPM (or even Node) at all here", maybe even opening up future possibilities to avoid having it be installed in the environment at all.

Reply:
Sounds good, should we keep those npm scripts moving forward? There might be value in it but I'm afraid that at some point they will diverge from what is added to Travis config.

Reply:
> Sounds good, should we keep those npm scripts moving forward? There might be value in it but I'm afraid that at some point they will diverge from what is added to Travis config.

Agreed on the concern. I know I've used the NPM script variants, likely out of convenience / familiarity of what would be the `docker-compose` equivalent.